atom/browser/lib/power_observer.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#ifndef ATOM_BROWSER_LIB_POWER_OBSERVER_H_
+#define ATOM_BROWSER_LIB_POWER_OBSERVER_H_
+
+#include "base/macros.h"
+
+#if defined(OS_LINUX)
+#include "atom/browser/lib/power_observer_linux.h"
+namespace atom {
+
+typedef PowerObserverLinux PowerObserver;
+
+}  // namespace atom
+#else
+#include "base/power_monitor/power_observer.h"
+namespace atom {
+
+typedef base::PowerObserver PowerObserver;
+
+}  // namespace atom
+#endif
+
+#endif  // ATOM_BROWSER_LIB_POWER_OBSERVER_H_
```

Comment:
Probably it would be simpler if the headers were separated from the rest of code as usual.

```cpp
#include "base/macros.h"
#if defined(OS_LINUX)
#include "atom/browser/lib/power_observer_linux.h"
#else
#include "base/power_monitor/power_observer.h"
#endif  // defined(OS_LINUX)

namespace atom {

#if defined(OS_LINUX)
typedef PowerObserverLinux PowerObserver;
#else
typedef base::PowerObserver PowerObserver;
#endif  // defined(OS_LINUX)

}  // namespace atom
```

Reply:
:+1: